### PR TITLE
[4.1.x] Allow date pickers to overflow and fix summary date picker

### PR DESCRIPTION
**Pull request by jordanwilking**
- Fixes an issue downstream with datepicker calendars in dialogs being rendered in the wrong place
- Fixes the summary datepicker calendar, which wouldn't allow time inputs

Testing:
- Verify date pickers works (field input, calendar input)
- Verify the summary tab datepicker works properly

### CODE REVIEW

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx [hunk 1]
```diff
@@ -36,6 +36,7 @@ import { displayHighlightedAttrInFull } from './highlightUtil'
 import DateTimePicker from '../../fields/date-time-picker'
 import Geometry from '../../../react-component/input-wrappers/geometry'
 import { useRerenderOnBackboneSync } from '../../../js/model/LazyQueryResult/hooks'
+import { truncate } from 'lodash'
```
Comment (jordanwilking) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx]: Don't know how this got in there. Removing it.

Comment (stevenmalmgren) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx]: Is this used?

**Review by stevenmalmgren — APPROVED** [review]

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx (hunk 1 above)
Comment (jordanwilking) [reply ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx]: No idea how it got in there, but nope. Removed.

**Comment by jlcsmith** [comment]
🎉 hero success 🎉

**Review by jlcsmith — APPROVED** [review]

**Pull request merged**